recipe-client-addon/lib/FilterExpressions.jsm

```diff
@@ -41,3 +43,23 @@ this.FilterExpressions = {
     return jexl.eval(onelineExpr, context);
   },
 };
+
+/**
+ * Return an array of the given object's own keys (specifically, its enumerable
+ * properties).
+ * @param {Object} obj
+ * @return {Array[String]}
+ */
+function keys(obj) {
+  return Object.keys(obj);
+}
+
+/**
+ * Find all the values that are present in both lists.
+ * @param {Array} listA
+ * @param {Array} listB
+ * @return {Array}
+ */
+function operatorIntersect(listA, listB) {
+  return listA.filter(item => listB.includes(item));
```

Comment:
I think this fails because `includes` isn't bound to `listB`. I tested in the web console:

```js
>> var list1 = [1, 2, 3]
undefined
>> var list2 = [2, 3, 4]
undefined
>> list1.filter(list2.includes)
TypeError: can't convert undefined to object
```